server/api/check_unique_url.py

```diff
@@ -23,6 +23,7 @@ def check_url_is_unique(user_id):
             unique_url = User.find_url(given_url)
             if unique_url:
                 output["error"] = "Unavailable"
+                status = 200
```

Comment:
status 400?

Reply:
Reason for 200 is that, API call is successful in both cases, whether we find unique url or not.